client/src/i18n/en.json

```diff
@@ -532,6 +539,14 @@
     "TITLE"          : "Bhima",
     "TODAY"          : "Date of day"
   },
+  "INVENTORY" : {
+    "ADD_GROUP"  : "Add inventory group",
+    "CONTAINS"   : "Contains",
+    "EDIT_GROUP" : "Edit inventory group",
+    "ELEMENT"    : "Elt",
```

Comment:
Elt seems like a pretty confusing abbreviation.  I wouldn't guess that "elt" was "element".